db/seeds/conversations.rb

```diff
@@ -16,3 +16,7 @@
 business = User.find_by(email: "part-time@example.com").business
 conversation = Conversation.find_or_create_by!(developer:, business:)
 SeedsHelper.create_message!(conversation:, sender: business, body: "Looking for freelance work?", cold_message: true)
+
+# Populate response rate for developers
+developers = Conversation.all.map(&:developer).uniq
```

Comment:
developers = Developer.joins(:conversations).distinct

Reply:
Thanks Joe. This helped me to finally fully understand what a JOIN table is!